source/tyr/tyr/__init__.py

```diff
@@ -49,6 +49,14 @@ def celery_setup_logging(*args, **kwargs):
     pass
 
 
+'''
+Since Celery 4.0, JSON is the default serializer replacing pickle.
+Problem is, some of our tasks are not JSON serializable (ie. class AutocompleteUpdateData)
+So back to pickle it is !
+'''
+app.config['CELERY_TASK_SERIALIZER'] = 'pickle'
```

Comment:
Maybe the actual fix for this would be to make sure the JSON serialisation works with all tasks like `AutocompleteUpdateData` etc... 

But that would mean, being sure we have tested them all not to crash in prod.